chainercv/chainer_experimental/training/extensions/make_shift.py

```diff
@@ -0,0 +1,41 @@
+from chainer.training import Extension
+
+
+def make_shift(attr, optimizer=None):
+    """Decorator to make shift extensions.
+
+    This decorator wraps a function and makes a shift extension.
+    Base function should takes :obj:`trainer` and returns a new value of
```

Comment:
should take :obj:`trainer` and return a ...
or 
takes :obj:`trainer` and returns a ...